test/docfx.Test/e2e/E2ETest.cs

```diff
@@ -65,15 +65,6 @@ private static async Task RunCore(string docsetPath, E2ESpec spec)
                 Assert.True(false, File.ReadAllText(Path.Combine(docsetOutputPath, "build.log")));
             }
 
-            // Verify restored files
-            foreach (var (file, content) in spec.Restores)
```

Comment:
There are also tests to check git clone location, do you still want those?

Reply:
yes, i just remove the lock file content check, but still want to make sure the dependencies are restored correctly

---
In reply to: [230305694](https://github.com/dotnet/docfx/pull/3570#discussion_r230305694) [](ancestors = 230305694)